### PR TITLE
Update Targets to include combined deaths processing

### PR DESCRIPTION
**Pull request by Jennit07**
After having a discussion about the processing flow, i have updated this to reflect the following: 

- Check the years specified in `targets.R`
- Run one year first to identify any potential bugs
- Work with another analyst to resolve any potential issues
- Once happy, check that all years specified in `targets.R` are correct
- One analyst can run the script `run_all_targets` to run the remaining years targets. ETA is tbc
- Once all targets have completed we can check through all of the tests for each of the extracts.
- Once we are happy, we can divide up the years between analysts and run the episode files. It is also recommended to run one year first to check for any potential bugs. 
- When we are happy with the episode files, we can then run the individual files. 

Summarised the process here but i will also update the SOP

### CODE REVIEW

**Review by lizihao-anu — DISMISSED** [review]

File: _targets.R [hunk 1]
```diff
@@ -542,106 +542,114 @@ list(
...
-    #     partitioning = c("hscp2018"),
-    #     compression = "zstd",
-    #     version = "latest"
-    #   )
-    # )
+    )
+  ),
+  tar_target(
+    combined_deaths_lookup,
+    process_combined_deaths_lookup()
```
Comment (lizihao-anu) [inline _targets.R:556]: I guess this needs to be moved out from targets.

Comment (SwiftySalmon) [reply _targets.R:556]: where does it go instead?

File: Run_SLF_Files_targets/run_all_targets.R [hunk 2]
```diff
@@ -0,0 +1,5 @@
+library(targets)
+
+# use tar_make_future() to run targets for all years
+# This will run everything needed for creating the episode file.
+tar_make_future()
```
Comment (lizihao-anu) [inline Run_SLF_Files_targets/run_all_targets.R:5]:
```suggestion
tar_make_future()

combined_deaths_lookup = process_combined_deaths_lookup()
```

File: _targets.R (hunk 1 above)
Comment (lizihao-anu) [reply _targets.R:556]: Sorry for not making it perfectly clear. We should move it to `run_all_targets.R`. By moving it, we can make sure that combining deaths lookup is after the death file for each year being produced. I add a code suggestion below.

**Review by SwiftySalmon — APPROVED** [review]
don't fully understand what is happening here but I trust yous.

**Pull request merged**